src/Service/InstallationService.php

```diff
@@ -45,7 +45,7 @@ public function createInstallation(Keypair $keypair): array
     /**
      * @return Id $deviceServerId
      */
-    public function createDeviceServer(Token $token, string $apiKey, string $description): Id
```

Comment:
I want to move to the `Token` class, and will update the example with the new instance